### ISSUE TITLE
3_2015 Test_Cycle_Social Icons

### ISSUE THREAD

**Issue by saracope** [BusinessUSA/BusinessUSA-Usability-Feedback#20]
Device Make: Apple
Device Model: iPhone 5s
Device Operating System Version: 7.1
Browser: Safari
Carrier (or WiFi): ATT
Severity: Low
Description:
When navigating to the homepage on mobile, the social icons are displayed (with odd numbers) at the top of the screen while the main site navigation is hidden in a hamburger menu. I think it would be more usable if this were reversed with the social links being hidden and the main nav displayed more prominently. 

![2015-03-05 09 39 07](https://cloud.githubusercontent.com/assets/2197515/6526530/f395ba5e-c3e0-11e4-938c-1b06088e985d.png)

**Comment by MagsStew** [comment]
I agree with this!  I felt like a top task for the user was share or "market" the site. I was surprised by the prominence of the social media icons.
